lib/libs/email/content/upload-subsequent-documents/emailTemplates/MedSpaCMS.tsx

```diff
@@ -0,0 +1,36 @@
+import { CommonEmailVariables, Events } from "shared-types";
+import {
+  PackageDetails,
+  BasicFooter,
+  Attachments,
+  SubDocHowToAccess,
+} from "../../email-components";
+import { BaseEmailTemplate } from "../../email-templates";
+
+export const MedSpaCMSEmail = (props: {
+  variables: Events["UploadSubsequentDocuments"] & CommonEmailVariables;
+}) => {
+  const variables = props.variables;
+  //subject
+  const previewText = `Medicaid SPA ${variables.id}`;
+  // start of body
+  const heading = `New documents have been submitted for Medicaid SPA ${variables.id} in OneMAC.`;
```

Comment:
Do you think we can move these variables to be declared inside the `BaseEmailTemplate` props? The comments seems a bit extra if the variables can be renamed

Reply:
The other emails don't seem to have comments either

Reply:
Will do, sorry I left these comments so I could pair with Thomas

Reply:
No problem! Happy to server as a second pair of eyes